docker/pwsh-exchange/Dockerfile

```diff
@@ -0,0 +1,5 @@
+
+FROM demisto/powershell-ubuntu:7.0.3.12001
+
+RUN pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted -ErrorAction Stop"
```

Comment:
No need for that, just disable GP before installation.
```suggestion
```

Reply:
@GalRabin It's not related to GP. Let's leave this. We add this automatically in our `create_new_docker_image.py` script. It is better to have this explicit.

Reply:
Ok